censys-subdomain-finder.py

```diff
@@ -10,95 +14,139 @@
 
 load_dotenv()
 
-NON_COMMERCIAL_API_LIMIT = 1000
+USER_AGENT = f"{CensysCerts.DEFAULT_USER_AGENT} (censys-subdomain-finder; +https://github.com/christophetd/censys-subdomain-finder)"
+
+MAX_PER_PAGE = 100
+COMMUNITY_MAX_PAGES = 10
+MAX_PAGES = 50
```

Comment:
Why 50? Shouldn't it be unlimited for the commercial plan?

Reply:
We can do unlimited if you want, but commercial plans still have quota limits. I just didn't want to max it out.

Reply:
Can we at least show a message if we know we truncate the results? Or allow overriding this through the CLI? wdyt?

Reply:
sounds good as long as they have a default  value

Reply:
Okay actually I went with the option of just making it unlimited, hopefully that works for you.